stagecraft/apps/datasets/tests/models/test_data_set.py

```diff
@@ -69,6 +72,31 @@ def test_saving_with_new_name_doesnt_change_it(self):
         data_sets = DataSet.objects.filter(name='data_group1_data_type1')
         assert_equal(len(data_sets), 1)
 
+    def test_can_create_dataset_with_owner(self):
+        DataSet.objects.create(
+            data_group=self.data_group1,
+            data_type=self.data_type1)
+        data_set1 = DataSet.objects.get(name='data_group1_data_type1')
+        user, _ = User.objects.get_or_create(
+            email='foobar.lastname@gov.uk')
+        data_set1.save()
+        data_set1.owners.add(user)
+
+        assert_equal('foobar.lastname@gov.uk', data_set1.owners.first().email)
+
+    def test_can_create_dataset_without_owner(self):
+        DataSet.objects.create(
+            data_group=self.data_group1,
+            data_type=self.data_type1)
+        data_set1 = DataSet.objects.get(name='data_group1_data_type1')
+        data_set1.save()
+
+        assert_equal(len(data_set1.owners.all()), 0)
+
+    def test_can_create_dataset_without_owner_in_forms(self):
```

Comment:
What we are really testing here is the behaviour that a dataset can be saved without an owner - we dont really mind about how it is implemented. There is another example of this [here](http://googletesting.blogspot.co.uk/2013/08/testing-on-toilet-test-behavior-not.html).

What would happen to this test if we changed the way owners were implemented in the future, and used a different relationship to the dataset to work out who the owners were?